frontend/src/context/BowtieVersionContext.jsx

```diff
@@ -0,0 +1,14 @@
+import { createContext, useMemo, useState } from "react";
+
+export const BowtieVersionContext = createContext();
+
+export const BowtieVersionContextProvider = ({ children }) => {
+  const [version, setVersion] = useState();
```

Comment:
Instead of creating a specific Context Provider for `version`, I was planning something like creating a general Context Provider for the entire `data`, in order to avoid prop drilling.

Reply:
Hmm, I needed access to version and couldn't use `useLoaderData` so I put it in context. If you think more general context is needed you are free to refactor this one in your later PRs, but honestly I don't know if putting everything in context is a good solution. But if you have any other ideas how to handle version i nav bar, please share